logfire/_internal/integrations/fastapi.py

```diff
@@ -58,7 +58,6 @@ def instrument_fastapi(
         dict[str, Any] | None,
     ]
     | None = None,
-    use_opentelemetry_instrumentation: bool = True,
```

Comment:
This option was a bad idea on my part, clutters the docs and API, and doesn't really have a purpose any more now that OTEL kwargs are passed through.